baselines/dom.generated.d.ts

```diff
@@ -2569,6 +2569,8 @@ interface Document extends Node, GlobalEventHandlers, NodeSelector, DocumentEven
       * @param content The text and HTML tags to write.
       */
     writeln(...content: string[]): void;
+    createElement(tagName: "picture"): HTMLPictureElement;
```

Comment:
these will be ignored if they are defined here after the less specific signature: `createElement(tagName: string)`. 
@zhengbli why is the order messed up here? and how can we fix it?